### PR TITLE
doc: Insert Arista license header in all source files

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Insert license header in all source files

## Proposed changes

Add pre-commit hook to insert license headers in:
- [x] Python files
- [x] Documentation files

Increase the scope of pre-commit hooks to include ./docs

The following files had additional fixes from pre-commit hooks 

```shell
trim trailing whitespace.......................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing docs/advanced_usages/as-python-lib.md
Fixing docs/cli/get-inventory-information.md
Fixing docs/getting-started.md
Fixing docs/cli/debug.md

fix end of files...............................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing docs/imgs/animated-svg.md
Fixing docs/snippets/anta_help.txt
Fixing docs/api/tests.system.md
Fixing docs/imgs/anta-nrfu.svg
Fixing docs/api/tests.md
Fixing docs/faq.md
Fixing docs/imgs/anta-nrfu.cast
```

### CODE REVIEW

File: license-short.txt
Comment (titom73) [inline license-short.txt:1]: Maybe we should move this file under `.github` folder as it is not a standard file for a repository ?

Comment (carlbuchmann) [reply license-short.txt:1]: yes, I will move it there!

Comment (carlbuchmann) [reply license-short.txt:1]: fixed in [a704d1d](https://github.com/arista-netdevops-community/anta/pull/367/commits/a704d1d39274a45c08566f1a37901f710879b2a3)

**Review by titom73 — APPROVED** [review]
LGTM! Thank you

**Pull request merged**